package.json

```diff
@@ -8,10 +8,12 @@
     "dev:bal": "turbo dev --filter frontend-v3",
     "dev:beets": "turbo dev --filter beets-frontend-v3",
     "graphql:gen": "turbo graphql:gen",
+    "lint:all": "turbo run lint format --continue",
+    "lint:all:fix": "turbo run lint:fix format:fix --continue --force",
     "lint": "turbo lint",
     "lint:fix": "turbo lint:fix",
-    "prettier": "prettier --check '**/*.*(md|json|yaml|ts|js|tsx)' --log-level warn",
-    "prettier:fix": "pnpm run prettier --write",
+    "format": "prettier '*.*(md|json|yaml|ts|js|tsx)' --check --cache --cache-location='node_modules/.cache/.prettiercache'",
+    "format:fix": "prettier '*.*(md|json|yaml|ts|js|tsx)' --write --cache --cache-location='node_modules/.cache/.prettiercache' --log-level=warn",
```

Comment:
Should this not be?

```suggestion
    "format": "turbo format",
    "format:fix": "turbo format:fix",
```

Reply:
this is needed to run prettier in the root (and only in the root)